dev-env/bin/ghcide

```diff
@@ -1,4 +1,9 @@
-#!/usr/bin/env bash
-DADE_CURRENT_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-source "$DADE_CURRENT_SCRIPT_DIR/../lib/dade-common"
-bazel build @ghcide-exe//ghcide && bazel-bin/external/ghcide/ghcide-0.7.2.0/_install/bin/ghcide "$@"
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ../../shell.nix
+
+set -e
+set -u
+set -x
```

Comment:
do we want this here?

Reply:
It only prints two lines and I found it helpful. I can remove it if you like.

Reply:
fine to keep it. Just seemed a bit odd to include it in this one place which doesn’t actually do anything particularly interesting while we don’t have it in most other scripts.